packages/fxa-payments-server/Gruntfile.js

```diff
@@ -9,10 +9,6 @@ module.exports = function (grunt) {
     // in a later ticket - will require coordination with l10n to resolve
     // conflicting IDs for identical terms.
     'src/branding.ftl',
-    // Adding shared branding file to allow for gradual adoption of shared
-    // branding terms. There are currently no conflicting IDs between the
-    // two branding.ftl files.
-    '../fxa-shared/l10n/branding.ftl',
```

Comment:
@bcolsson `fxa-shared/l10n/branding.ftl` was removed, and the branding file local to this project was kept instead.